generator/php/resources/templates/AnalyticsExample.mustache

```diff
@@ -4,9 +4,9 @@
 // Then import it as follow:
 // require_once(__DIR__ . '/vendor/autoload.php');
 
-use Criteo\SDK\Api\AnalyticsApi;
-use Criteo\SDK\Model\StatisticsReportQueryMessage;
-use Criteo\SDK\TokenAutoRefreshClient;
+use {{invokerPackage}}Api\AnalyticsApi;
+use {{invokerPackage}}Model\StatisticsReportQueryMessage;
+use {{invokerPackage}}TokenAutoRefreshClient;
 
 /*
  * Although the OpenAPI specification, then this generated client, you can't simply use the API key feature.
```

Comment:
Line 25: it should be `ClientCredentialsClient` instead of `TokenAutoRefreshClient`